From: SeongJae Park <sj@kernel.org>
Date: Wed, 7 Jan 2026 17:42:50 -0800
Subject: [PATCH] ==== docs updates ====

Docs/mm/damon: update intro, modules, maintainer profile, and misc

Update DAMON documentations for wordsmithing, clarifications, and
miscellaneous outdated things with eight patches.  Patch 1 simplifies
the brief introduction of DAMON.  Patch 2 updates DAMON user-space tool
packaged distros information on design doc to include not only Fedora,
but refer to repology.  Three following patches update design and usage
documents for clarifying DAMON sample modules purposes (patch 3), and
outdated information about usages of DAMON modules (patches 4 and 5).
Final three patches update usage and maintainer-profile for sysfs
refresh_ms feature behavior (patch 6), synchronize DAMON MAINTAINERS
section name (patch 7), and broken damon-tests performance tests
(patch 8).

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 hkml_cv_bogus/hkml_cv_bogus_xyjl6k14 | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 hkml_cv_bogus/hkml_cv_bogus_xyjl6k14

diff --git a/hkml_cv_bogus/hkml_cv_bogus_xyjl6k14 b/hkml_cv_bogus/hkml_cv_bogus_xyjl6k14
new file mode 100644
index 000000000000..e69de29bb2d1
-- 
2.47.3

